src/compile-cache.js

```diff
@@ -179,9 +179,10 @@ exports.install = function(resourcesPath, nodeRequire) {
         return null;
       }
 
-      let match, lastMatch;
+      let lastMatch;
       INLINE_SOURCE_MAP_REGEXP.lastIndex = 0;
-      while ((match = INLINE_SOURCE_MAP_REGEXP.exec(fileData))) {
+      let match = INLINE_SOURCE_MAP_REGEXP.exec(fileData);
+      while (match) {
         lastMatch = match;
```

Comment:
```suggestion
        lastMatch = match;
        match = INLINE_SOURCE_MAP_REGEXP.exec(fileData);
```

Same as above. Alternatively a do-while loop might be better, although the ordering is backwards because of the later `if (lastMatch == null)`:
```js
let match, lastMatch;
INLINE_SOURCE_MAP_REGEXP.lastIndex = 0;
do {
  lastMatch = match;
  match = INLINE_SOURCE_MAP_REGEXP.exec(fileData);
} while (match)
```

Reply:
Same mistake as above. Committed your suggestion. Tomorrow will recheck them all more accurately